logstash/README.md

```diff
@@ -0,0 +1,42 @@
+# Loki Logstash Output Plugin
+Logstash plugin to send logstash aggregated logs to Loki.
+
+
+## Install dependencies
+First you need to setup JRuby environment to build this plugin. Refer https://github.com/rbenv/rbenv for setting up your rbenv environment.
+
+After setting up `rbenv`. Install JRuby
+```
+rbenv install jruby-9.2.10.0
```

Comment:
Do we want to align with the fluentD plugin on how to install and build the plugin. Think use of rbenv, vanilla ruby vs jruby. Is jruby required for this plugin to function?

Reply:
Logstash uses JRuby

Reply:
I want to extend something here.

We should have a pre baked docker image with that plugin installed.

Reply:
👍